packages/shadydom/src/patches/Element.js

```diff
@@ -120,6 +120,23 @@ export const ElementPatches = utils.getOwnPropertyDescriptors({
       this[utils.NATIVE_PREFIX + 'removeAttribute'](attr);
     }
   },
+
+  /**
+   * @this {Element}
+   * @param {string} attr
+   * @param {boolean | undefined} force
+   */
+  toggleAttribute(attr, force) {
+    if (this.ownerDocument !== doc) {
+      this[utils.NATIVE_PREFIX + 'toggleAttribute'](attr, force);
```

Comment:
```suggestion
      return this[utils.NATIVE_PREFIX + 'toggleAttribute'](attr, force);
```
(and on the others below)